build_config/lib/src/build_target.g.dart

```diff
@@ -8,13 +8,8 @@ part of 'build_target.dart';
 
 BuildTarget _$BuildTargetFromJson(Map json) {
   return $checkedNew('BuildTarget', json, () {
-    $checkKeys(json, allowedKeys: const [
-      'builders',
-      'dependencies',
-      'sources',
-      'key',
```

Comment:
`key` and `package` are properties on `BuildTarget`, but they are not set-able via the ctor – so they shouldn't be allowed.

Same below...